conf/deployment/ibmcloud/ibm_cloud_vpc_cluster.yaml

```diff
@@ -4,22 +4,23 @@
 ENV_DATA:
   platform: 'ibm_cloud'
   deployment_type: 'managed'
-  region: 'eu-de'
-  zone: 'eu-de-3'
+  region: 'us-south'
+  zone: 'us-south-2'
   provider: "vpc-gen2"
-  worker_instance_type: "mx2.16x128"
+  worker_instance_type: "bx2.16x64"
   worker_replicas: 3
+  volname : 'voltest'
```

Comment:
You should not change those values as this is something which in QE are using . If you need to use something else, please create custom config file on your side or create another production config file.  Or just create some config file under https://github.com/red-hat-storage/ocs-ci/tree/6ecc96ca640e96448a16e2271f5553a7a6248c12/conf/ocsci where you will have only specific changes you need and will use often. Like different region or zone.